scripts/tests/chiptest/yamltest_with_chip_repl_tester.py

```diff
@@ -46,13 +46,14 @@
     __ALL__ = (matter_idl_types)
 
 
+import click
+from matter_yamltests.definitions import SpecDefinitionsFromPaths
+from matter_yamltests.parser import PostProcessCheckStatus, TestParser, TestParserConfig
```

Comment:
same with `matter_yamltests`, should it be considered first party?

Reply:
Same as above. Maybe it should be `matter.yamltests`?

Reply:
I think `matter.*` would be better. Unsure if we used the current naming to satisfy our build rules ... if we do not have to fight it very hard, I prefer `matter.testing` and `matter.yamltests` and such.